### PR TITLE
Simplify table names in ETL management

### PR DESCRIPTION
**Pull request by calina-c**
Fixes # .

Changes proposed in this PR:

- 
- 
-

### CODE REVIEW

File: pdr_backend/lake/table.py [hunk 1]
```diff
@@ -141,9 +130,66 @@ def __init__(self, table_name: str, table_type: TableType = TableType.NORMAL):
...
+        return TempTable(dataclass.get_lake_table_name())  # type: ignore[attr-defined]
+
+    @staticmethod
+    # type: ignore[override]
+    # pylint: disable=arguments-differ
+    def from_table(table: Table) -> "TempTable":
+        return TempTable(table.dataclass.get_lake_table_name())  # type: ignore[attr-defined]
+
+
+class EtlTable(NamedTable):
```
Comment (idiom-bytes) [inline pdr_backend/lake/table.py:181]: Can we rename this to ETL?

File: pdr_backend/lake/table.py [hunk 2]
```diff
@@ -141,9 +130,66 @@ def __init__(self, table_name: str, table_type: TableType = TableType.NORMAL):
...
+    def __init__(self, table_name: str):
+        super().__init__(table_name, TableType.ETL)
+
+    @staticmethod
+    # type: ignore[override]
+    # pylint: disable=arguments-differ
+    def from_dataclass(dataclass: type) -> "EtlTable":
+        return EtlTable(dataclass.get_lake_table_name())  # type: ignore[attr-defined]
```
Comment (idiom-bytes) [inline pdr_backend/lake/table.py:189]: Same

File: pdr_backend/lake/table.py [hunk 3]
```diff
@@ -141,9 +130,66 @@ def __init__(self, table_name: str, table_type: TableType = TableType.NORMAL):
...
+
+    @staticmethod
+    # type: ignore[override]
+    # pylint: disable=arguments-differ
+    def from_table(table: Table) -> "EtlTable":
+        return EtlTable(table.dataclass.get_lake_table_name())  # type: ignore[attr-defined]
```
Comment (idiom-bytes) [inline pdr_backend/lake/table.py:195]: Same

File: pdr_backend/lake/table.py (hunk 1 above)
Comment (calina-c) [reply pdr_backend/lake/table.py:181]: ETLTable instead of EtlTable?

**Review by idiom-bytes — CHANGES_REQUESTED** [review]
Only request is to rename Etl -> ETL

**Review by idiom-bytes — APPROVED** [review]
Looks great! Ty for the updates!

**Pull request merged**